go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/k8snetworkplumbingwg/accelerated-bridge-cni
 go 1.13
 
 require (
-	github.com/Mellanox/sriovnet v1.0.1
+	github.com/Mellanox/sriovnet v1.0.3
 	github.com/containernetworking/cni v0.8.0
 	github.com/containernetworking/plugins v0.8.7
 	github.com/onsi/ginkgo v1.12.0
 	github.com/onsi/gomega v1.9.0
 	github.com/rs/zerolog v1.20.0
 	github.com/spf13/afero v1.6.0
-	github.com/stretchr/testify v1.5.1
-	github.com/vishvananda/netlink v1.1.0
+	github.com/stretchr/testify v1.7.0
+	github.com/vishvananda/netlink v1.1.1-0.20210518155637-4cb3795f2ccb
```

Comment:
both sriovnet and sriov-network-device-plugin use v1.1.1-0.20211101163509-b10eb8fe5cf6

why did you take this specific version ?

Reply:
Not sure how it happened during the automatic update, switched to the common version for sriovnet and device-plugin

Reply:
Not sure how it happened during the automatic update, switched to the common version for sriovnet and device-plugin

Reply:
I did not update to the latest sriovnet version initially - after switching to v1.1.0 the netlink version is also consistent with sriovnet and device plugin